examples/default/example.tf

```diff
@@ -2,13 +2,35 @@ provider "aws" {
   region = "eu-west-1"
 }
 
-data "aws_vpc" "main" {
-  default = true
-}
+locals {
+  name_prefix    = "vpn-example"
+  vpc_cidr_block = "10.10.0.0/16"
 
-data "aws_subnet_ids" "main" {
-  vpc_id = "${data.aws_vpc.main.id}"
+  tags = {
+    terraform   = "True"
+    environment = "example"
+  }
 }
 
-# REST OF THE EXAMPLE
+module "vpc" {
+  source               = "telia-oss/vpc/aws"
+  version              = "0.2.0"
+  name_prefix          = "${local.name_prefix}"
+  cidr_block           = "${local.vpc_cidr_block}"
+  private_subnet_count = "0"
+  create_nat_gateways  = "false"
+  enable_dns_hostnames = "true"
+  tags                 = "${local.tags}"
+}
 
+module "vpn" {
+  source                          = "../../"
+  customer_gateway_bgp_asn        = "64952"
+  customer_gateway_ip             = "53.36.74.73"
```

Comment:
What is this ip?

Reply:
Ah yes... that was a random number I pulled out of the air.  It has to be valid public IP address for the example to work.  I guess I should comment that and put a more recognisable IP address there - perhaps 8.8.8.8?

Reply:
Maybe we should use an ip from here? https://tools.ietf.org/html/rfc5737